elasticapm/instrumentation/packages/dbapi2.py

```diff
@@ -121,6 +121,11 @@ def scan(tokens):
 
 
 def extract_signature(sql):
```

Comment:
Not part of this PR, but out of interest, why is this not a method of `CursorProxy` ? Afaics this is only used within sql cursor proxy implementations.

Reply:
I thought about this as well, but into the other direction: remove `CursorProxy.extract_signature` and call `dbapi2.extract_signature()` directly where we need it.

I guess the idea was that if we instrument a SQL backend that needs a different parser, we could plug it in there. @roncohen might remember.

Reply:
Thank you for giving some context. Might be worth giving it a thought at some point, but as mentioned unrelated to this PR.